### PR TITLE
[CONTENT] Newsletter 100 - January 2021 news

### PR DESCRIPTION
**Pull request by Extravert-ir**

### CODE REVIEW

File: content/blogs/newsletter-100.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+title:       "Newsletter 100 - January 2021 news"
+author:      "Extravert34"
+date:        2021-02-06
+tags:        [ newsletter ]
+---
+
+Hello ReactOS followers!
+This is an attempt to restore the tradition of annual newsletters covering our resent development efforts.
```
Comment (binarymaster) [inline content/blogs/newsletter-100.md:9]:
```suggestion
This is an attempt to restore the tradition of annual newsletters covering our recent development efforts.
```

File: content/blogs/newsletter-100.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+The previous [Newsletter 99](/blogs/newsletter-99) was in 2013!
+
+This month we were mostly focused on the kernel work, merging pull requests and fixing regressions
```
Comment (binarymaster) [inline content/blogs/newsletter-100.md:12]:
```suggestion
This month we were mostly focused on the kernel work, merging pull requests and fixing regressions.
```

File: content/blogs/newsletter-100.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+Being in development for almost half a year, this work by Jérôme Gardou finally landed in the master branch.
+This change refactors many aspects of Section Objects feature in NT-compatible kernel, making it more compatible with Windows.
+
+Here are notable changes from the user perspective:
+
+* A long-standing bug preventing to start executables from remote locations is fixed ([CORE-10038](https://jira.reactos.org/browse/CORE-10038)).
+* Open source Microsoft FAT filesystem driver is enabled by default now.
+* Better compatibility with FS drivers in general, `ntfs.sys` and `exfat.sys` drivers from Windows are now close to working state in ReactOS.
+* External drives with FAT filesystem can now be properly ejected
```
Comment (binarymaster) [inline content/blogs/newsletter-100.md:24]:
```suggestion
* External drives with FAT filesystem can now be properly ejected.
```

File: content/blogs/newsletter-100.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+## Changes for GCC amd64 build
+
+At the moment, we support building for amd64 target only with MSVC compiler.
+Hervé Poussineau fixed most of the places preventing the compilation with GCC, but there are still some linker issues not fixed.
+
+## NTFS boot sector
+
+Tests show that our homegrown NTFS driver is capable to boot ReactOS (with experimental features enabled),
+and `ntfs.sys` driver from Windows is also close to working state. So it's the right time for working on a bootloader for this filesystem.
+The Freeloader already supports working with NTFS, only the VBR bootsector part is missing.
```
Comment (binarymaster) [inline content/blogs/newsletter-100.md:35]:
```suggestion
The FreeLoader already supports working with NTFS, only the VBR boot sector part is missing.
```

File: content/blogs/newsletter-100.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+Sylvain Deverre started to work on the support in a [pull request](https://github.com/reactos/reactos/pull/3416).
+We need more people to test different configurations with this VBR, testers and contributors are welcome!
+
+After this VBR work, the only missing bit for supporting NTFS installation (with both our and proprietary driver)
+would be a formatting tool in the ReactOS installer.
```
Comment (binarymaster) [inline content/blogs/newsletter-100.md:41]:
```suggestion
would be formatting support in the ReactOS installer and in the shell.
```

File: content/blogs/newsletter-100.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+## PnP and I/O kernel subsystems
+
+A major code cleanup and refactoring was made here by Victor Perevertkin,
+bringing better compatibility with Windows drivers and **booting from USB media**.
+
+It's still not perfect though (some hardware we tested has issues with USB controller),
+but now you can just dump a nightly build of ReactOS onto a flash drive and try to boot/install it.
+If it didn't work for you check out other installing options available on our [wiki](https://reactos.org/wiki/Installing_ReactOS#Installing_ReactOS)
```
Comment (binarymaster) [inline content/blogs/newsletter-100.md:50]:
```suggestion
If it didn't work for you check out other installing options available on our [wiki](https://reactos.org/wiki/Installing_ReactOS#Installing_ReactOS).
```

File: content/blogs/newsletter-100.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+This change also improves debugging capabilities, which will speed up driver bugs fixing in future.
+
+## Kernel-mode Driver Framework
+
+Max Korostil continued [the work](https://github.com/reactos/reactos/pull/3276) on a `wdfldr.sys` driver for loading a KMDF library. This will eventually allow ReactOS to have KMDF drivers
+in its codebase, without hack we currently use for static linking.
```
Comment (binarymaster) [inline content/blogs/newsletter-100.md:57]:
```suggestion
in its codebase, without the hack we currently use for static linking.
```

File: content/blogs/newsletter-100.md (hunk 1 above)
Comment (ColinFinck) [reply content/blogs/newsletter-100.md:9]: Also "annual" = "yearly". You probably mean "recurring".

File: content/blogs/newsletter-100.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+Hello ReactOS followers!
+This is an attempt to restore the tradition of annual newsletters covering our resent development efforts.
+The previous [Newsletter 99](/blogs/newsletter-99) was in 2013!
+
+This month we were mostly focused on the kernel work, merging pull requests and fixing regressions
+
+## Memory manager and Common cache work
+
+Being in development for almost half a year, this work by Jérôme Gardou finally landed in the master branch.
```
Comment (Doug-Lyons) [inline content/blogs/newsletter-100.md:16]: Suggested change. "half a year" to "six months".

File: content/blogs/newsletter-100.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+This change refactors many aspects of Section Objects feature in NT-compatible kernel, making it more compatible with Windows.
```
Comment (Doug-Lyons) [inline content/blogs/newsletter-100.md:17]: Suggestion. "of Section Objects" to "of the Section Objects"

File: content/blogs/newsletter-100.md [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+
+Here are notable changes from the user perspective:
+
+* A long-standing bug preventing to start executables from remote locations is fixed ([CORE-10038](https://jira.reactos.org/browse/CORE-10038)).
```
Comment (Doug-Lyons) [inline content/blogs/newsletter-100.md:21]: Suggestion "preventing to start" to "prevented starting of"

File: content/blogs/newsletter-100.md [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+* Open source Microsoft FAT filesystem driver is enabled by default now.
+* Better compatibility with FS drivers in general, `ntfs.sys` and `exfat.sys` drivers from Windows are now close to working state in ReactOS.
```
Comment (Doug-Lyons) [inline content/blogs/newsletter-100.md:23]: Suggestsion: "close to working state" to "close to a working state".

File: content/blogs/newsletter-100.md [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+* External drives with FAT filesystem can now be properly ejected
+
+## Changes for GCC amd64 build
+
+At the moment, we support building for amd64 target only with MSVC compiler.
```
Comment (Doug-Lyons) [inline content/blogs/newsletter-100.md:28]: Suggestsion: At the moment, we support building for an amd64 target only with the MSVC compiler.

File: content/blogs/newsletter-100.md [hunk 13]
```diff
@@ -0,0 +1,64 @@
...
+Hervé Poussineau fixed most of the places preventing the compilation with GCC, but there are still some linker issues not fixed.
+
+## NTFS boot sector
+
+Tests show that our homegrown NTFS driver is capable to boot ReactOS (with experimental features enabled),
+and `ntfs.sys` driver from Windows is also close to working state. So it's the right time for working on a bootloader for this filesystem.
```
Comment (Doug-Lyons) [inline content/blogs/newsletter-100.md:34]: Suggestion: "close to working state" to "close to a working state", and "So it's the right time" to "So it's a good time"

File: content/blogs/newsletter-100.md [hunk 14]
```diff
@@ -0,0 +1,64 @@
...
+
+Sylvain Deverre started to work on the support in a [pull request](https://github.com/reactos/reactos/pull/3416).
+We need more people to test different configurations with this VBR, testers and contributors are welcome!
+
+After this VBR work, the only missing bit for supporting NTFS installation (with both our and proprietary driver)
+would be a formatting tool in the ReactOS installer.
+
+## PnP and I/O kernel subsystems
+
+A major code cleanup and refactoring was made here by Victor Perevertkin,
```
Comment (Doug-Lyons) [inline content/blogs/newsletter-100.md:45]: Suggestioon: "made here" to "done here"

File: content/blogs/newsletter-100.md [hunk 15]
```diff
@@ -0,0 +1,64 @@
...
+bringing better compatibility with Windows drivers and **booting from USB media**.
```
Comment (Doug-Lyons) [inline content/blogs/newsletter-100.md:46]: Suggestion: "bringing" to "allowing"

File: content/blogs/newsletter-100.md [hunk 16]
```diff
@@ -0,0 +1,64 @@
...
+
+It's still not perfect though (some hardware we tested has issues with USB controller),
```
Comment (Doug-Lyons) [inline content/blogs/newsletter-100.md:48]: Suggestion: "controller" to "controllers"

File: content/blogs/newsletter-100.md (hunk 6 above)
Comment (Doug-Lyons) [inline content/blogs/newsletter-100.md:50]: Suggestion: "If it didn't" to "If it doesn't"

**Pull request merged**